src/content/docs/en/pages/main-menu/compliance/soc-2.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: SOC Compliance
+description: Azion receives annual SOC 2 Type 2 and SOC 3 reports, which demonstrate that the security controls are continuously audited over the course of the year.
+meta_tags: Azion, edge computing, compliance, data security and privacy
+namespace: documentation_compliance_soc
+permalink: /documentation/compliance/soc/
+---
+
+Azion consistently strives to enhance internal security controls to assure customers that the applications, content, and data running on Azion Edge Computing Platform are kept safe and available. In this tradition, Azion receives annual **SOC 2 Type 2** and **SOC 3** reports, which demonstrate that the security controls are continuously audited over the course of the year.
+
+---
+
+## What are SOC 2 Type II and SOC 3?
+
+**SOC 2** is an auditing procedure developed by the **American Institute of Certified Public Accountants (AICPA)** to ensure that service providers are taking the necessary steps to maintain data security and privacy. SOC 2 attests to the trustworthiness of services provided by the organization, and it's composed of five “trust service principles” that serve as a backbone for secure data management.
+
+---
+
+## Azion's evaluation
+
+Azion is evaluated on two different trust principles: **Security** and **Availability**.
+
+This third-party evaluation assesses the platform's ability to prevent unauthorized access, security breaches, and security-related issues that might impact availability.
+
+Azion's **SOC 2 Type 2** attestation applies to the entire Azion Edge Computing Platform, which includes the Edge Application product and its modules (Application Accelerator, Edge Cache, Edge Functions, and Load Balancer) and the Edge Firewall product and its modules (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF, and Origin Shield), covering all regions where it's deployed. Azion invested effort into attesting the entire Platform for mainly one reason: performance. Flawless experiences demand that apps and data are processed and delivered as close to end users as possible, and certifying less than the entire network and Platform would lead to a performance tradeoff.
```

Comment:
```suggestion
Azion's **SOC 2 Type 2** attestation applies to the entire Azion Edge Computing Platform, which includes the Edge Application product and its add-ons (Application Accelerator, Edge Cache, Edge Functions, and Load Balancer) and the Edge Firewall product and its add-ons (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF, and Origin Shield), covering all regions where it's deployed. Azion invested effort into attesting the entire Platform for mainly one reason: performance. Flawless experiences demand that apps and data are processed and delivered as close to end users as possible, and certifying less than the entire network and Platform would lead to a performance tradeoff.
```